doc/content/source/bcs/SakiyamaSecondaryElectronBC.md

```diff
@@ -1,20 +1,38 @@
 # SakiyamaSecondaryElectronBC
 
-!alert construction title=Undocumented Class
-The SakiyamaSecondaryElectronBC has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
 !syntax description /BCs/SakiyamaSecondaryElectronBC
 
 ## Overview
 
-!! Replace these lines with information regarding the SakiyamaSecondaryElectronBC object.
+`SakiyamaSecondaryElectronBC` is an electron mean energy density of secondary electrons induced by ion flux outflow boundary condition.
+`SakiyamaSecondaryElectronBC` assumes the electrostatic approximation for the electric field.
+
+The ion induce secondary electron mean energy density outflow is defined as
+
+\begin{equation}
+a =
+\begin{cases}
+1, & \mu_{j} \ \text{-} \nabla (V) \cdot \textbf{n} > 0\\
+0, & \mu_{j} \ \text{-} \nabla (V) \cdot \textbf{n} \leq 0\\
+\end{cases} \\[10pt]
+\Gamma_{e} \cdot \textbf{n} = \gamma \left[a \ \mu_{j} \ \text{-} \nabla (V) \cdot \textbf{n} \ n_{j} \right]
+\end{equation}
+
+Where $\Gamma$ is the outflow normal to the boundary, $\textbf{n}$ is the normal of the boundary,
+$\mu_{j}$ is the mobility coefficient, $n_{j}$ is the ion density, $\gamma$ is the secondary electron coefficient, and $V$ is the potential. $a$ is defined such that the outflow is only defined when the drift velocity is direct towards the wall and zero otherwise. When converting the density to log form and applying a scaling factor of the mesh, the strong form for `SakiyamaSecondaryElectronBC` is defined as
```

Comment:
```suggestion
Where $\Gamma_e \cdot \textbf{n}$ is the flux normal to the boundary, $\textbf{n}$ is the normal vector of the boundary,
$\mu_{j}$ is the mobility coefficient, $n_{j}$ is the ion density, $\gamma$ is the secondary electron coefficient, and $V$ is the electric potential. $a$ is defined such that the outflow is only non-zero when the drift velocity is directed towards the wall and zero otherwise. When converting the density to log form and applying a scaling factor of the mesh, the strong form for `SakiyamaSecondaryElectronBC` is defined as
```